cli/cmd/up/up.go

```diff
@@ -20,6 +20,7 @@ type Up struct {
 	Branch             string   `desc:"Set branch when pointing stack to git repo" default:"master"`
 	BuildCloneSecret   string   `desc:"Set git clone secret name"`
 	BuildWebhookSecret string   `desc:"Set GitHub webhook secret name"`
+	PushRegistrySecret string   `desc:"Set secret for pushing to custom registry"`
```

Comment:
Lets add this to the cli ref docs too: https://github.com/rancher/rio/blob/master/docs/cli-reference.md#up